main.c

```diff
@@ -29,14 +29,15 @@
 
 // The bootloader can be entered in three ways:
 //  - BOOTLOADER_ENTRY_PIN is low
-//  - Watchdog scratch[5] == BOOTLOADER_ENTRY_MAGIC && scratch[6] == ~BOOTLOADER_ENTRY_MAGIC
+//  - Watchdog scratch[5] == BOOTLOADER_ENTRY_PIN && scratch[6] == ~BOOTLOADER_ENTRY_MAGIC
```

Comment:
This `BOOTLOADER_ENTRY_PIN` is changed here but `BOOTLOADER_ENTRY_MAGIC` is still used in function `should_stay_in_bootloader`